### PR TITLE
Add Setting UI and insert record into db on Finish page

### PR DESCRIPTION
**Pull request by huishi329**

### CODE REVIEW

File: react-app/src/Finish/index.js [hunk 1]
```diff
@@ -4,8 +4,37 @@ import Header from './Header'
 import Image from './Image'
 import Buttons from './Buttons'
 import Lottie from 'lottie-react-native'
+import { useEffect, useState } from 'react'
+import * as SQLite from 'expo-sqlite'
 
 export default function Finish() {
+  const db = SQLite.openDatabase('shelter.db')
```
Comment (j3rryzh) [inline react-app/src/Finish/index.js:11]: I think we've already had a database object `db` which is from prop.

File: react-app/src/Finish/index.js [hunk 2]
```diff
@@ -4,8 +4,37 @@ import Header from './Header'
...
+  const [debugMessage, setDebugMessage] = useState('')
+
+  useEffect(() => {
+    const timestamp = new Date().getTime()
+    db.transaction(
+      (tx) => {
+        tx.executeSql(
+          `insert into Record (timestamp, exercise_name) values (?,?);`,
```
Comment (j3rryzh) [inline react-app/src/Finish/index.js:19]: The table have 3 column but just insert 2 value here, might cause an error.

File: react-app/src/Finish/index.js (hunk 1 above)
Comment (huishi329) [reply react-app/src/Finish/index.js]: Fixed!

File: react-app/src/Finish/index.js (hunk 2 above)
Comment (huishi329) [reply react-app/src/Finish/index.js:18]: I think this is alright! Thank you for double checking!

**Review by mylesieong — APPROVED** [review]

**Review by akaka0039 — APPROVED** [review]

**Pull request merged**